docs/document/docs/components/desktop/button.md

```diff
@@ -26,6 +26,7 @@ Here is a list of properties that can be used for modifying the component:
 | className | string | ""  | Component class name |  |
 | id | string | ""  | Component id name |  |
 | text | string | ""  | Text to be displayed inside the button | |
+| content | string/HTMLElement | ""  | The DOM inside button | If a string with HTML is set, it will be automatically converted to HTML and displayed as it is<br>The value of `text` will be displayed when the `content` is as follows: <ul><li>undefined</li><li>null</li><li>Empty string</li></ul>In other cases, the `text` property will be ignored |
```

Comment:
How about arranging it like this?
After the confirmation, let's update the spec description, too.

```suggestion
| text | string | ""  | Text to be displayed inside the button | If `content` is unspecified, the value of `text` will be displayed<br>In other cases, the `text` will be ignored  |
| content | string/HTMLElement | ""  | The DOM inside button | If a string with HTML is set, it will be automatically converted to HTML and displayed as it is |
```

Reply:
There is one case as below
The default value of `content` is "".
If the developer assigns an empty string or null/undefined to `content`, the `text` will also be ignored.

Reply:
We haven't announced the info that we distinguish null/undefined/empty strings yet, so I think it's ok to explain with "unspecified" word that includes the meaning of null/undefined/empty string.
How do you think?

And please correspond to this comment, too.
> Please update the sample code section of the user guide.
> It's ok with the shorthand version of icon for example ....

Reply:
Got it.
It's ok now.